mavutil.py

```diff
@@ -432,7 +435,7 @@ def post_message(self, msg):
         elif type == 'PARAM_VALUE':
             if not src_tuple in self.param_state:
                 self.param_state[src_tuple] = param_state()
-            self.param_state[src_tuple].params[msg.param_id] = msg.param_value
+            self.param_state[src_tuple].params[msg.param_id.decode('ascii')] = msg.param_value
```

Comment:
Probably it makes sense to also add the error here or index the params by bytes internally.

Reply:
Also note that in python2 decode produces a unicode value not a string value, so this might also cause potential issues.